packages/vkui/src/components/Slider/SliderThumb/SliderThumb.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import { classNames } from '@vkontakte/vkjs';
+import { useFocusVisible } from '../../../hooks/useFocusVisible';
+import type { HasRootRef } from '../../../types';
+import { FocusVisible } from '../../FocusVisible/FocusVisible';
+import styles from './SliderThumb.module.css';
+
+export type SliderThumbEventPayload = {
+  min: number;
+  max: number;
+  value: number;
+};
+
+interface SliderThumbProps
+  extends Omit<React.HTMLAttributes<HTMLSpanElement>, 'onKeyDown'>,
+    HasRootRef<HTMLSpanElement> {
+  min: number;
+  value: number;
+  max: number;
+  disabled?: boolean;
+  onKeyDown?(event: React.KeyboardEvent<HTMLSpanElement>, payload: SliderThumbEventPayload): void;
+}
+
+export const SliderThumb = ({
+  min,
+  value,
+  max,
+  className,
+  disabled,
+  getRootRef,
+  onKeyDown,
+  ...restProps
+}: SliderThumbProps) => {
+  const { focusVisible, onBlur, onFocus } = useFocusVisible(false);
+
+  const handleKeyDown: React.KeyboardEventHandler<HTMLSpanElement> = (event) => {
+    if (onKeyDown) {
+      onKeyDown(event, { min, max, value });
+    }
+  };
+
+  return (
+    <span
+      ref={getRootRef}
+      tabIndex={disabled ? undefined : 0}
+      role="slider"
+      aria-valuemin={min}
+      aria-valuenow={value}
+      aria-valuemax={max}
+      aria-disabled={disabled}
+      className={classNames(
+        styles['SliderThumb'],
+        !disabled && styles['SliderThumb--notDisabled'],
+        className,
+      )}
+      onKeyDown={disabled ? undefined : handleKeyDown}
+      onBlur={disabled ? undefined : onBlur}
+      onFocus={disabled ? undefined : onFocus}
+      {...restProps}
+    >
+      <FocusVisible
```

Comment:
А что, если прокидывать текстовые метки для лучшей доступности так:
```suggestion
    >
      <VisuallyHidden>{children}</VisuallyHidden>
      <FocusVisible
```

Reply:
А `aria-label` плохо считываются? В гайдах и в других библиотеках используется именно он

`aria-label` и по реднедеру эффективней, т.к. не создаётся разметка в DOM и не надо CSS обрабатывать

Reply:
На самом деле всегда лучше, чтобы контент был в самом элементе, потому что aria-метки штука довольно специфическая и не всегда может считываться правильно теми же разными скринридерами.))) Поэтому по возможности хорошо бы от них уйти в пользу скрытого текста в самом компоненте.

Reply:
Я переделал на нативный `<input type="range" />`, теперь должно срабатывать очевидно, как в доке https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/slider_role завешают)

Предлагаю VisualHidden применять если мы идём в обход нативным элементам и обвешиваемся `div`/`span`, что думаешь?